data/campaigns/Sceptre_of_Fire/scenarios/9_Caverns_of_Flame.cfg

```diff
@@ -88,10 +89,21 @@
             [goal]
                 name=target
                 [criteria]
-                    side=2,3
+                    side=2,6
                 [/criteria]
                 value=1
             [/goal]
+            [goal]
+                name=lua_goal
+                engine=lua
+                code = <<
+                    local turn = wesnoth.current.turn
+                    local targets = {}
+                    targets[1] = { value = 2, type = 2, loc = { x = 29, y = 7 } }
+                    targets[2] = { value = turn * 2 , type = 2, loc = { x = 19, y = 17 } }
```

Comment:
Please use strings for the target type in both of these lines (in this case that would be `type = 'explicit'`).